sdk/core/azure-core/src/http/retry_policy.cpp

```diff
@@ -31,7 +31,7 @@ bool GetResponseHeaderBasedDelay(RawResponse const& response, Delay& retryAfter)
     return true;
   }
 
-  if ((header = responseHeaders.find("Retry-After")) != responseHeadersEnd)
+  if ((header = responseHeaders.find("retry-after")) != responseHeadersEnd)
```

Comment:
LGTM!  Thank you!
@antkmsft  the transport policy would call `toLower` to all headers read from the network.